lib/intercept.js

```diff
@@ -123,6 +123,11 @@ function remove(interceptor) {
 }
 
 function removeAll() {
+  Object.keys(allInterceptors).forEach(function(key) {
+    allInterceptors[key].scopes.forEach(function(interceptor) {
```

Comment:
Since we're accessing so many nested properties could we add some defensive coding here? Might also be nice to add `debug` log statements if the properties are unavailable